srcpkgs/ulogd/files/ulogd.logrotate

```diff
@@ -1,9 +1,9 @@
-/var/log/ulog.log /var/log/ulog.pcap {
-    missingok
+/var/log/ulogd/* {
```

Comment:
Please use a more exact wildcard or set of wildcards. E.g. `/var/log/ulogd/*.log /var/log/ulogd/*.pcap`. See below:

>       Please use wildcards with caution.  If you specify *, logrotate will rotate all files, including previously rotated ones.  A way around this is to use the olddir directive or a more exact wildcard (such as    *.log).

https://man.voidlinux.org/logrotate.8

Reply:
Good catch. Thanks.